src/Trace/Tracer/ExtensionTracer.php

```diff
@@ -261,6 +271,14 @@ private function getKind($span)
         return Span::KIND_UNSPECIFIED;
     }
 
+    private function getSameProcessAsParentSpan($span)
+    {
+        if (method_exists($span, 'sameProcessAsParentSpan')) {
+            return $span->sameProcessAsParentSpan();
+        }
+        return true;
```

Comment:
When this happens? and is it ok to always return true?

Reply:
This would happen if they have an older version of the extension. 

We could try to do a workaround to try and set the first span's value to false, but it's probably not worth it.